libkbfs/folder_branch_ops.go

```diff
@@ -333,7 +333,13 @@ var _ fbmHelper = (*folderBranchOps)(nil)
 // newFolderBranchOps constructs a new folderBranchOps object.
 func newFolderBranchOps(config Config, fb FolderBranch,
 	bType branchType) *folderBranchOps {
-	nodeCache := newNodeCacheStandard(fb)
+	var nodeCache NodeCache
+	if config.Mode() != InitMinimal {
```

Comment:
I prefer if possible for `if`/`else` statements to have the positive case first.